ReSharper.FSharp/src/FSharp.Psi.Services/src/Daemon/Highlightings/FcsErrors.xml

```diff
@@ -92,6 +92,7 @@
     <QuickFix>ReplaceWithWildPatFix</QuickFix>
     <QuickFix>RemoveUnusedNamedAsPatFix</QuickFix>
     <QuickFix>AddUnderscorePrefixFix</QuickFix>
+    <QuickFix>RemoveUnusedNamedPatternFix</QuickFix>
```

Comment:
The quick fix is currently at the end of the list but is arguably much more useful than `ReplaceWithWildPatFix` and `AddUnderscorePrefixFix `. Let's move it up? 🙂